build/import/HostAgnostic.props

```diff
@@ -16,15 +16,17 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="WindowsBase" />
 
+    <!-- Infrastructure -->
+    <PackageReference Include="MicroBuild.Core.Sentinel" />
+    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" />
```

Comment:
Created an `Infrastructure` block since these packages are no longer only related to the RepoToolset. As RepoToolset usage is reduced/removed, likely more packages would go into this section as they will likely run natively with MicroBuild plugins. Currently, most packages in the `Toolset` section use `ExcludeAssets="all"` which means they don't run via the plugins as the assets don't actually interact with the projects. Instead, the packages get restored but custom code in RepoToolset accesses them directly. Things in this `Infrastructure` section will properly use the package contents.

Reply:
My understanding of these is that anyone who references our packages will now inherit a reference to these packages too. Should these be `PrivateAssets="all"`?

Reply:
Good find! https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

That's something that wouldn't have been obvious to me from simply building everything through the infrastructure. I wonder if there is a way we could test for that... For now, I've added this to those elements. Thanks!